modulus/models/module.py

```diff
@@ -197,6 +197,7 @@ def save(self, file_name: Union[str, None] = None, verbose: bool = False) -> Non
             local_path = Path(temp_dir)
 
             torch.save(self.state_dict(), local_path / "model.pt")
+            print("save path: ", local_path, "/model.pt")
```

Comment:
Can these print statements be added directly to the example scripts for training and inference?

Reply:
sorted